### PR TITLE
Use run group and update Dockerfile

### PR DESCRIPTION
**Pull request by pbalogh-sa**
- update Dockerfile
- use run group

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -82,7 +78,7 @@ bin/licensei-${LICENSEI_VERSION}:
 
 .PHONY: license-check
 license-check: bin/licensei ## Run license check
-	bin/licensei check
+	bin/licensei check cmd
```
Comment (sagikazarmark) [inline Makefile]:
```suggestion
	bin/licensei check
```

File: .licensei.toml [hunk 2]
```diff
@@ -9,4 +9,5 @@ approved = [
 ignored = [
   "github.com/gogo/protobuf",
   "sigs.k8s.io/yaml",
+  "github.com/oklog/run",
```
Comment (sagikazarmark) [inline .licensei.toml]: Is this really necessary? Apache 2.0 license is approved.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,85 +1,143 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
```
Comment (sagikazarmark) [inline .circleci/config.yml]:
```suggestion
version: 2

```

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,85 +1,143 @@
+# Golang CircleCI 2.0 configuration file
```
Comment (sagikazarmark) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 5]
```diff
@@ -1,85 +1,143 @@
...
+#
```
Comment (sagikazarmark) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 6]
```diff
@@ -1,85 +1,143 @@
...
+# Check https://circleci.com/docs/2.0/language-go/ for more details
```
Comment (sagikazarmark) [inline .circleci/config.yml]:
```suggestion
```

File: .licensei.toml (hunk 2 above)
Comment (pbalogh-sa) [reply .licensei.toml]:
```
github.com/oklog/run: no license file found
make: *** [license-check] Error 2
```

**Pull request merged**